### DISCUSSION TITLE
Using ConfigMaps

### DISCUSSION THREAD

**Discussion by theprobugmaker** [okteto/okteto#3009] [Q&A] [answered]
I'm trying to use the Okteto Stacks to run a few applications and most of those applications have config files that need to be loaded using ConfigMaps, how can I do that with Okteto `okteto deploy`? Thank you.

**Comment by pchico83** [comment, marked as answer]
@theprobugmaker currently the only way to load k8s configmaps in your application is to use kubernetes manifests or helm charts, configmaps is not supported by the okteto docker compose integration
